docs/openapi/components/schemas/common/SC237AForm.yml

```diff
@@ -0,0 +1,193 @@
+$linkedData:
+  term: SC237AForm
+  '@id': https://w3id.org/traceability#SC237AForm
+title: SC-237A Form
```

Comment:
Two suggestions: 
1. Please prefix with `USDA`
2. Spell out "SC" to make this less esoteric. 

I would suggest naming this `USDASpecialtyCrops237AForm`.

Reply:
@nissimsan I just made this change, could you mark this request as complete so it doesn't block the merge?

Reply:
Excellent